dts/bindings/regulator/nxp,vref.yaml

```diff
@@ -20,12 +20,6 @@ properties:
   reg:
     required: true
 
-  nxp,ground-select:
```

Comment:
This commit needs to be after the change dropping the `gnd_sel` field from the driver

Reply:
yes